src/main/java/de/uhd/ifi/se/decision/management/jira/rest/KnowledgeRest.java

```diff
@@ -151,6 +151,10 @@ public Response createDecisionKnowledgeElement(@Context HttpServletRequest reque
 		existingElement.setDocumentationLocation(documentationLocationOfExistingElement);
```

Comment:
this is not necessary, please remove these lines because you overwrite the object later on